src/bin/pg_dump/pg_dump.c

```diff
@@ -8562,6 +8562,21 @@ getCasts(Archive *fout, int *numCasts)
 		/* Decide whether we want to dump it */
 		selectDumpableCast(&(castinfo[i]), fout);
 
+		/*
```

Comment:
I assume this can impact PG dump. Could there be a case where a PG database has cast from pg_catalog to sys (sys could be user defined schema). We need to keep in mind which change can break PG dump restore or not. It seems this one can. We have to think of alternatives so that we do this logic only when dumping a babel DB.

Reply:
Yeah added logic to check if the object is part of bbf extensions.